src/components/usa-link/index.js

```diff
@@ -26,19 +26,18 @@ export class UsaLink extends LitElement {
     const childLink = this.querySelector("a");
     if (!childLink) return false;
 
-    this.href = childLink.href;
     this.slottedChildren = childLink;
+    this.slottedChildren.classList.add('usa-link');
     this.shadowRoot.appendChild(this.slottedChildren);
+    return true;
```

Comment:
Without this line, I was observing the slot template is always rendered and the child template is never rendered.